pinot-tools/src/main/java/org/apache/pinot/tools/admin/PinotAdministrator.java

```diff
@@ -192,7 +192,11 @@ public static void main(String[] args) {
     PluginManager.get().init();
     PinotAdministrator pinotAdministrator = new PinotAdministrator();
     pinotAdministrator.execute(args);
-    if (System.getProperties().getProperty("pinot.admin.system.exit", "false").equalsIgnoreCase("true")) {
+    // Ignore `pinot.admin.system.exit` property for Pinot quickstarts.
+    if ((args.length > 0) && ("quickstart".equalsIgnoreCase(args[0]))) {
+      System.clearProperty("pinot.admin.system.exit");
+    }
+    if (Boolean.parseBoolean(System.getProperties().getProperty("pinot.admin.system.exit"))) {
       // If status is true, cmd was successfully, so return 0 from process.
       System.exit(pinotAdministrator._status);
```

Comment:
why not add it here:
```suggestion
      if (args.length == 0 || !"quickstart".equalsIgnoreCase(args[0])) {
        System.exit(pinotAdministrator._status);
      }
```

Reply:
it require a check for property `pinot.admin.system.exit`.
Also commands like `StartController/Broker/Server/Minion` don't need this as well.